openmc/deplete/stepresult.py

```diff
@@ -198,6 +199,34 @@ def distribute(self, local_materials, ranges):
         new.rates = [r[ranges] for r in self.rates]
         return new
 
+    def get_material(self, mat_id):
```

Comment:
Perhaps this new ```StepResut.get_material``` is quite similar to ```Result.export_to_materials``` in some cases.

However they operate in different ways.

I am just wondering if there might be an opportunity for ```Result.export_to_materials``` to call ```StepResut.get_material``` and save some lines of code.

Reply:
Agreed, there is semi-duplication here. If you don't mind, let's defer this for later -- I'm planning on doing some refactoring with materials which should simplify things here (and elsewhere).

Reply:
Sounds good to me.